src/SearchParameters/RefinementList.js

```diff
@@ -74,8 +74,8 @@ var lib = {
    * @param {string} value the value of the refinement
    * @return {RefinementList} a new and updated list
```

Comment:
No. This is the abstract set of functions that handle the structures that manages the facets. Actually this should not even be renamed (see removeRefinement). In this context, it is "clear" that we are not dealing with anything else than facets.